src/App.tsx

```diff
@@ -1,14 +1,46 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [todoId, setTodoId] = useState(0);
+  const [statusFilter, setStatusFilter] = useState('all');
+  const [textFilter, setTextFilter] = useState('');
+
+  useEffect(() => {
+    getTodos()
+      .then(response => {
```

Comment:
Let’s use `async await` all over the code and in future test tasks too. It gives you a simpler way to work with asynchronous promise-based code.